Sources/SwiftSyntaxBuilder/Buildables.swift.gyb

```diff
@@ -103,8 +103,9 @@ public struct ${node.syntax_kind}: ${node.base_kind}Buildable {
   public init(
 %     init_parameters = []
 %     for child in node.children:
+%       child_token = SYNTAX_TOKEN_MAP.get(child.syntax_kind)
 %       param_type = syntax_buildable_child_type(child.type_name, child.syntax_kind, child.is_token(), child.is_optional)
-%       default_value = " = nil" if child.is_optional else ""
+%       default_value = syntax_buildable_default_init_value(child, child_token) # " = nil" if child.is_optional else ""
```

Comment:
Is the `# " = nil" if child.is_optional else ""` still a comment left over from development?